pkg/lib/userexport/model.go

```diff
@@ -22,6 +22,7 @@ const BatchSize = 1000
 
 var defaultCSVExportFields = []*FieldPointer{
 	&FieldPointer{Pointer: "/sub"},
+	&FieldPointer{Pointer: "/created_at"},
```

Comment:
For this list, it is better to group them into standard claims, and our proprietary claims, separated by a newline.

```
&FieldPointer{Pointer: "/sub"},
&FieldPointer{Pointer: "/preferred_username"},
...

&FieldPointer{Pointer: "/roles"},
...

```